mlrun/runtimes/base.py

```diff
@@ -1425,11 +1415,33 @@ def _encode_requirements(self, requirements_to_encode):
 
         return " ".join(requirements)
 
+    def _validate_output_path(self, run):
+        if is_local(run.spec.output_path):
```

Comment:
what if that is empty?

Reply:
empty is considered local and means that no artifact path was set at any level and it will be enriched to a local path.